### DISCUSSION TITLE
Is there a way to create a reorderable checkbox list?

### DISCUSSION THREAD

**Discussion by th0r** [SBoudrias/Inquirer.js#1396] [Q&A] [answered]
Hi!

I need to allow user to pick items from the list **and** allow to reorder them. Is there a way to do so? Thanks!

**Comment by SBoudrias** [comment, marked as answer]
Hi, this use case being quite specific, it's a great case to [create a custom prompt](https://github.com/SBoudrias/Inquirer.js#create-your-own-prompts).

You can copy-paste the current [checkbox implementation](https://github.com/SBoudrias/Inquirer.js/blob/main/packages/checkbox/src/index.mts) as a starting point.

**Reply by th0r** [reply]
Ok, thanks!

**Reply by SBoudrias** [reply]
If you end up creating one and open sourcing it, it'd be great to link it in [our community prompt list over here](https://github.com/SBoudrias/Inquirer.js#community-prompts).

**Reply by th0r** [reply]
Created a custom prompt for this case: https://github.com/th0r/inquirer-sortable-checkbox

PR with readme update: https://github.com/SBoudrias/Inquirer.js/pull/1400
